Ical.Net/Calendar.cs

```diff
@@ -238,11 +244,17 @@ public virtual HashSet<Occurrence> GetOccurrences(DateTime startTime, DateTime e
     /// <param name="dt">The date for which to return occurrences. Time is ignored on this parameter.</param>
     /// <returns>A list of Periods representing the occurrences of this object.</returns>
     public virtual HashSet<Occurrence> GetOccurrences<T>(IDateTime dt) where T : IRecurringComponent
-        => GetOccurrences<T>(new CalDateTime(dt.Date), new CalDateTime(dt.Date.AddDays(1).AddTicks(-1)));
```

Comment:
Now that CalDateTime is rounded to seconds, ticks aren't helpful anymore. Question would be generally, whether subtracting an 'Epsilon' is the best idea. It would certainly be cleaner to get the comparisons later on right, i.e. to compare for endTime to be exclusive, rather than inclusive.